lib/apiservers/engine/backends/container.go

```diff
@@ -934,347 +967,6 @@ func (c *Container) ContainerAttach(name string, ca *backend.ContainerAttachConf
 	return err
 }
 
-//-------------------------------------
```

Comment:
Assuming this is clean up -- just moving chairs around on the deck?  if so why included here and not a PR specifically addressing that goal?